distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/common/downloads/SynchronousClientDownloader.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.test.itests.common.downloads;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import org.apache.http.nio.client.methods.HttpAsyncMethods;
+
+/**
+ * Provides an asynchronous handle to the synchronous retrieval and download endpoint of a product.
+ */
+class SynchronousClientDownloader extends ClientDownloader {
+    private final InterruptableAsyncCharConsumer consumer = new InterruptableAsyncCharConsumer();
+
+    private final StringBuilder resultsBuilder = new StringBuilder();
```

Comment:
If we don't want to cache the results from the closed client using some form of consumer in `org.apache.http` then this can be removed.